src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+function createDiv(divClass, message) {
+  const div = document.createElement('div');
+
+  div.className = divClass;
+  div.textContent = message;
+
+  return div;
+}
+
+function successHandler(message) {
+  const successDiv = createDiv('message', message);
+
+  document.body.append(successDiv);
+}
+
+function errorHandler(message) {
+  const errorDiv = createDiv('error-message', message);
+
+  document.body.append(errorDiv);
+}
+
+const promise1 = new Promise(resolve => {
+  logo.addEventListener('click', () => resolve('Promise was resolved!'));
+});
+
+const promise2 = new Promise(
+  (resolve, reject) => setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000));
+
+promise1.then(successHandler);
+
+promise2.catch(errorHandler);
```

Comment:
_add success and error handlers for both promises_ - In this scenerio your promises can either resolve or reject. But normally when working with promises we are always trying to act on success and on error.

Try to add success and error handlers for both promises